jr-objects/src/main/java/com/fasterxml/jackson/jr/ob/impl/ValueLocatorBase.java

```diff
@@ -61,6 +61,9 @@ public abstract class ValueLocatorBase
     public final static int SER_INT_ARRAY = 5;
     public final static int SER_LONG_ARRAY = 6;
     public final static int SER_BOOLEAN_ARRAY = 7;
+    public final static int SER_SHORT_ARRAY = 38;
```

Comment:
Needs to be consecutive.

Reply:
... which gets tricky wrt whether to renumber entries or move these to later.

I think renumbering is needed, so move `SER_TREE_NODE` etc up by number of added entries needed.

Reply:
Yes, I saw that, and had the same thought, but I figured you'd know which of the two options (renumbering all later constants, or moving the other arrays to the end, which is ugly) would make the most sense.